### ISSUE TITLE
configToSpec debug log is noisy

### ISSUE THREAD

**Issue by jmrodri** [automationbroker/bundle-lib#148]
```
time="2018-07-23T21:27:58Z" level=debug msg="adapter::configToSpec -> Got plans [{ID: Name:default Description:Default deployment plan for kubevirt-apb using existing Metadata:map[longDescription:This plan deploys kubevirt cost:$0.00 displayName:Default] Free:true Bindable:false Parameters:[{Name:admin_user Title:OpenShift Admin User Type:string Description: Default:<nil> DeprecatedMaxlength:0 MaxLength:0 MinLength:0 Pattern: MultipleOf:0 Maximum:<nil> ExclusiveMaximum:<nil> Minimum:<nil> ExclusiveMinimum:<nil> Enum:[] Required:true Updatable:false DisplayType: DisplayGroup: Dependencies:[]} {Name:admin_password Title:OpenShift Admin Password Type:string Description: Default:<nil> DeprecatedMaxlength:0 MaxLength:0 MinLength:0 Pattern: MultipleOf:0 Maximum:<nil> ExclusiveMaximum:<nil> Minimu     m:<nil> ExclusiveMinimum:<nil> Enum:[] Required:true Updatable:false DisplayType:password DisplayGroup: Dependencies:[]} {Name:version Title:Version Type:enum Description: Default:0.7.0 DeprecatedMaxlength:0 MaxLength:0 MinLength:0 Pattern: MultipleOf:0 Maximum:<nil>  ExclusiveMaximum:<nil> Minimum:<nil> ExclusiveMinimum:<nil> Enum:[0.7.0 0.6.0 0.5.0 0.4.0 0.3.0 0.2.0 0.1.0] Required:false Updatable:false DisplayType: DisplayGroup: Dependencies:[]} {Name:registry_url Title:Registry URL Type:string Description: Default:docker.io DeprecatedMaxlength:0 MaxLength:0 MinLength:0 Pattern: MultipleOf:0 Maximum:<nil> ExclusiveMaximum:<nil> Minimum:<nil> ExclusiveMinimum:<nil> Enum:[] Required:false Updatable:false DisplayType: DisplayGroup: Dependencies:[]} {Name:registry_namespace Title:Registry Nam     espace Type:string Description: Default:kubevirt DeprecatedMaxlength:0 MaxLength:0 MinLength:0 Pattern: MultipleOf:0 Maximum:<nil> ExclusiveMaximum:<nil> Minimum:<nil> ExclusiveMinimum:<nil> Enum:[] Required:false Updatable:false DisplayType: DisplayGroup: Dependencies:[]}] BindParameters:[] UpdatesTo:[]}]"
```

This is just ONE entry.

**Comment by jmrodri** [comment]
Out of a 1.4M log file, 1.1M is configSpec

```
[jesusr@speed3 config{master}]$ ls -l *.log
-rw-rw-r--. 1 jesusr jesusr 1.4M Jul 23 18:11 asb.log
-rw-rw-r--. 1 jesusr jesusr 1.1M Jul 23 19:03 configtospec.log
```

**Comment by jmrodri** [comment]
[configtospec.log.gz](https://github.com/automationbroker/bundle-lib/files/2221375/configtospec.log.gz)

**Comment by jmrodri** [comment]
Propose we remove it the log entirely.

**Comment by jmrodri** [comment]
Fixed by PR #151
